packages/blockchain-api/src/blockscout.ts

```diff
@@ -27,6 +28,19 @@ export interface BlockscoutCeloTransfer {
   value: string
 }
 
+// Calculate cGLD to cUSD rate based on the actual transfers
+function getImpliedCeloToCUSDExchangeRate(
+  current: BlockscoutCeloTransfer,
+  comparable: BlockscoutCeloTransfer
+): Pick<MoneyAmount, 'impliedCeloToCUSDExchangeRate'> {
+  return {
+    ...(current.token === CGLD &&
+      comparable.token === CUSD && {
+        impliedCeloToCUSDExchangeRate: new BigNumber(comparable.value).dividedBy(current.value),
+      }),
+  }
+}
+
```

Comment:
nit: syntax seems a bit confusing at first, why not just returning the value (or undefined)?
I guess you wanted to avoid having the key in the final object, right?

Reply:
Yeap, to avoid having `undefined` in the final object.